product_end_of_life_dates/models/product_template.py

```diff
@@ -8,4 +8,6 @@ class ProductTemplate(models.Model):
 
     end_of_support = fields.Date()
     end_of_sale = fields.Date()
-    end_of_life = fields.Date()
\ No newline at end of file
+    end_of_life = fields.Date()
```

Comment:
There seems to a line break after this field definition. Can you remove it?

Reply:
I don't understand, should there be an empty line after this field definition, or what exactly needs changing?